packages/injectable/react/src/getInjectableComponent/getInjectableComponent.test.jsx

```diff
@@ -654,6 +654,55 @@ describe('getInjectableComponent', () => {
         .discovered,
     ).toBeInTheDocument();
   });
+
+  it('given an injectable component with a placeholder and is the implementation of some injection token > when the injecting the component using the token > when the component suspends > shows the placeholder', () => {
+    const di = createContainer('some-container-1');
+    const someFn = asyncFn();
```

Comment:
Motivation of this is to never resolve, therefore:
1. Use of `asyncFn` is misleading
2. Variable name `someFn` should indicate that. Suggested: `const eternalPromise = new Promise(resolve => {});`.